hyperdash_cli/cli.py

```diff
@@ -261,12 +269,22 @@ def _is_port_open(host, port):
 
 
 def login(args=None):
-    email = get_input("Email address: ")
-    password = get_input("Password: ", True)
-    success, default_api_key = _login(email, password)
-    if success:
-        print("Successfully logged in! We also installed: {} as your default API key".format(
-            default_api_key))
+    if not (args.email or args.github):
+        print("To login with your email address, run `hd login --email`. Alternatively, you can login via Github by running `hd login --github")
+        return
+
+    if args.email:
+        email = get_input("Email address: ")
+        password = get_input("Password: ", True)
+        success, default_api_key = _login(email, password)
+        if success:
+            print("Successfully logged in! We also installed: {} as your default API key".format(
+                default_api_key))
+        return
+    
+    if args.github:
+        github(args)
+        return
 
```

Comment:
Any reason to style the control flow this way as opposed to `if/elif/else`?

Reply:
generally try to avoid unnecessary else's